.github/DEVELOPMENT.md

```diff
@@ -0,0 +1,134 @@
+# Development
+
+Developers should read [the development section of the website](https://trino.io/development),
+which covers things like development philosophy and contribution process.
+
+* [Getting started](#getting-started)
+* [Running tests](#running-tests)
+* [Commits and pull requests](#commits-and-pull-requests)
+* [Code style](#code-style)
+* [Releasing](#releasing)
+
+## Getting started
+
+Start by forking the repository and then modify the code in your fork.
+
+We recommend that you use Python3's `venv` for development:
+
+```
+$ python3 -m venv .venv
+$ . .venv/bin/activate
```

Comment:
nit: first dot redundant

Reply:
Also I remember someone from the docs team advising that everything that can't be copy-pasted should be omitted from a code block, like the starting `$`.

Reply:
the first dot isn't redundant - the activate from venv needs to be "sourced" and cannot be executed (at-least by default since it's not executable).

Thanks for that pointer @mdesmet - removed the `$`. I think it's useful in some cases to clarify what is input and what is output and "what prompt" is something being executed in. In this file however there's no ambiguity and it looks much better now.